index.html

```diff
@@ -6024,6 +6031,9 @@ <h4>JsonLdErrorCode</h4>
         <dt><dfn>invalid vocab mapping</dfn></dt>
         <dd>An invalid <a>vocabulary mapping</a> has been detected,
           i.e., it is neither an <a>IRI</a> nor <code>null</code>.</dd>
+        <dt class="changed">iri confused with prefix</dt>
+        <dd>When compacting an <a>absolute IRI</a> would result in an <a>absolute IRI</a>
+          with an <a data-cite="RFC3986#section-3.1">IRI scheme</a> could be confused with a term.</dd>
```

Comment:
```suggestion
          which could be confused with a compact IRI (because its <a data-cite="RFC3986#section-3.1">IRI scheme</a> matches a <a>term definition</a> and it has no <a data-cite="RFC3986#section-3.1">IRI authority</a>).</dd>
```
* "which" was missing from the original sentence
* this phrasing is in line with my proposed change above